lib/examples/compare-chromedp/remote/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/ysmood/rod"
+)
+
+var flagDevToolWsUrl = flag.String("devtools-ws-url", "", "DevTools WebSsocket URL")
```

Comment:
CI failed at here. Seems like a lint issue. You can run `godev lint` at the root folder to check. About how to install godev you can check the CI settings: `.github/workflows/test.yml`